workshop/exercise-3/README.md

```diff
@@ -1,79 +1,64 @@
-## Setup: Install Istio and Knative on Your Cluster
+## Deploy our First Application to Knative using the Knative Client (kn)
+A relatively new project, the knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple node.js with express app which returns the first n numbers of the Fibonacci sequence. To use the app, deploy it, and simply make a GET request to the `/` endpoint with a number as the parameter. 
 
-Knative is currently built on top of both Kubernetes and Istio.
-If you want to learn more about Kubernetes or Istio, you can check out the
-labs [Kube101](https://github.com/IBM/kube101/tree/master/workshop) and
-[Istio101](https://github.com/IBM/istio101/tree/master/workshop).
-When you install Knative on IKS, it will install Istio for you
-automatically.
+# Our First Knative Service
+Knative Serving supports deploying and serving of serverless applications and functions. Those applications and functions will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
```

Comment:
Seems like we should be a bit more consistent. In here you use "service", "application" and "function" interchangeably and that might confuse people.  I like

Reply:
Agree, removed function - left it as "application" since we later say that "services" represent applications.